File: 412-fizz-buzz/fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        FBA = []
        for i in range(1,n+1):
            if ( i % 3 == 0 and i % 5 == 0):
                FBA.append("FizzBuzz")
            elif (i % 3 ==0):
                FBA.append("Fizz")
            elif (i % 5 == 0):
                FBA.append("Buzz")
            else:
                FBA.append(str(i))
        return FBA
        

File: 412-fizz-buzz/fizz-buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    FBA = [];
    for(let i = 1;i<=n;i++){
        if (i % 3 == 0 && i % 5 == 0)
        FBA.push("FizzBuzz");
        else if(i % 3 == 0)
        FBA.push("Fizz");
        else if(i % 5 == 0)
        FBA.push("Buzz");
        else
        FBA.push(String(i))
    }
    return FBA;
};